docs/knowledge_base/semantic_model.zh.md

```
@@ -0,0 +1,179 @@
+## 语义模型如何增强 SQL 生成
+
+生成 SQL 时，agent 使用语义模型来：
+
+1. **Schema Linking**：使用描述的向量搜索找到相关表和列
+2. **正确的 JOIN 构造**：使用实体关系生成正确的 JOIN 条件
+3. **智能过滤**：应用使用模式（例如，对逗号分隔的标签列使用 `FIND_IN_SET()`）
+4. **准确的聚合**：为聚合函数选择适当的度量
+
+查询流程示例：
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)